src/style.css

```diff
@@ -0,0 +1,76 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big_bar,
+.small_bar {
+  font-family: inherit;
```

Comment:
ok, but I think better to explicitly set font

Reply:
added font-family: Avenir, sans-serif.